### PR TITLE
signed the md document

### PR DESCRIPTION
**Pull request by cbear1988uk**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,4 +39,4 @@ Now add yours!
 
 ---
 
-
+### Cb34r was here
```
Comment (mikelbrierly) [inline README.md:42]: Hey Collin! Great job with this code here! 

Our of curiosity, what does the "#" sign do?

Comment (cbear1988uk) [reply README.md:42]: Hi Mikel,

the # sign takes a line of txt and increases the font size and weight

Regards,
Collin

**Comment by mikelbrierly** [comment]
Approved!

**Comment by samleemcknight** [comment]
Wowwwwww

**Pull request merged**